src/data/python-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+   {
+    Question:
+      "In Python, which of the following option will reverse a_string='good' and make it 'doog'?",
+    Answer: "a_string[::-1]",
+    Distractor1: "reversed(a_string)",
+    Distractor2: "a_string.reverse()",
+    Distractor3: "a_string.sort(reverse=True)",
+    Explanation:
+      "<string>[start: stop: step] returns a slice of the string. <string>[:::] returns a copy of the entire string. <string>[::-1] returns a reversed copy of the string.",
```

Comment:
Let's simplify this a bit.

```suggestion
      "In Python, [::-1] will return a reversed copy of the string.",
```